packages/common/auth/src/types/authmaterial.ts

```diff
@@ -1,10 +1,28 @@
 import type { SDKExternalUser } from "./user";
 
+export type AuthMaterialBasic = {
+    jwtToken?: string;
+    refreshToken: string;
+};
+
+interface refreshTokenObject {
```

Comment:
two small things: 1. first letter uppercase, 2. "object" doesn't add meaning 
```suggestion
interface RefreshToken {
```